apps/HelloAndroidCamera2/build.gradle

```diff
@@ -0,0 +1,168 @@
+import org.apache.tools.ant.taskdefs.condition.Os
+
+repositories {
+    jcenter()
+}
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:1.0.0'
+    }
+
+}
+
+////////////////////////////////////////////////////////////////////////////////
+// Use gradle's native C++ plugin to build the halide generator.
```

Comment:
Not yet. gradle now has rudimentary support for automatically generating its own Android.mk file, but it's really limited. Not good enough to get rid of the .mk files yet.